test/common.coffee

```diff
@@ -2,7 +2,7 @@ mm = require '../src/mongodb-migrations'
 mongoConnect = require('../src/utils').connect
 
 config =
-  host: 'localhost'
+  host: process.env.DB_HOST || 'localhost'
```

Comment:
Are these tests ever run on CI?

Reply:
> Are these tests ever run on CI?

We don't have CircleCI running anything for this project. I don't think the original author had CI set up, either.